app/addfriend.tsx

```diff
@@ -8,17 +8,35 @@ import axios from "axios";
 export default function AddFriend() {
   const [userId, setUserId] = useState<string>();
   const [addFriendUrl, setAddFriendUrl] = useState<string>("");
+  const [userLang, setUserLang] = useState("fa");
+  const [translatedData, setTranslatedData] = useState<any>([]);
+
+  useEffect(() => {
+    (async () => {
+      await axios
+        .post<{ userLang: string }>(
```

Comment:
Looks great, but we have to find a way to call it at just one spot and then pass the translated data around. It would be an optimization task if we get time.